cmd/dev.go

```diff
@@ -103,6 +108,11 @@ Starts your project in development mode:
 	devCmd.Flags().BoolVar(&cmd.SwitchContext, "switch-context", true, "Switch kubectl context to the DevSpace context")
 	devCmd.Flags().BoolVar(&cmd.ExitAfterDeploy, "exit-after-deploy", false, "Exits the command after building the images and deploying the project")
 
+	devCmd.Flags().StringVarP(&cmd.Interactive, "interactive", "i", interactiveDefaultPickerValue, "Enable interactive mode for images (overrides entrypoint with sleep command) and start terminal proxy")
```

Comment:
I'm not sure if we should name the flag interactive, since the major change is that an image will be rebuild using a different entrypoint. I prefer 'sleep' as an option, but we should talk about this.

Reply:
I thought about that as well but `sleep` has some issues:
- It is very confusing because of the sleep mode for spaces. 
- It is too implementation-specific. We are using the sleep command for now but we could also do something like `cat` or `tail -f /dev/null` instead. They do the same but are not related to sleep.
- Additionally, the terminal is also enabled to having a name that expresses more than just build-related terminology makes sense.

The flag should be named after what the user wants to achieve with it. `interactive` expresses that the user has to do something and a terminal session is as interactive as it gets. The only alternative I could think of is `manual` because it says that the user has to manually start the container. That would make sense as well but there is not much of a difference between `manual` and `interactive`. More a matter of preference, I think. I would be ok with both of them.